apps/mobile/src/components/browser/browser-sheet/search-bar/toolbar-button.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactNode } from 'react';
+
+import { Pressable, Text, legacyTouchablePressEffect } from '@leather.io/ui/native';
```

Comment:
Is there a `legacyTouchablePressEffect` and a new version? Maybe we can just have a new one. I don't understand the `legacy` part?